payments-core/src/test/java/com/stripe/android/payments/financialconnections/GetFinancialConnectionsAvailabilityTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.stripe.android.payments.financialconnections
+
+import com.stripe.android.core.utils.FeatureFlags.financialConnectionsFullSdkUnavailable
+import com.stripe.android.core.utils.FeatureFlags.financialConnectionsLiteKillswitch
+import com.stripe.android.model.ElementsSession
+import com.stripe.android.model.ElementsSession.Flag.ELEMENTS_DISABLE_FC_LITE
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class GetFinancialConnectionsAvailabilityTest {
+
+    @Test
+    fun `when full SDK available and not unavailable should return full`() {
+        financialConnectionsFullSdkUnavailable.setEnabled(false)
```

Comment:
Nit: Can you use `FeatureFlagTestRule` here?

Reply:
https://github.com/stripe/stripe-android/pull/10504/commits/788dda569279d2997c45d54bd968871d5dbb0d1e